dev/registry.sh

```diff
@@ -182,6 +187,22 @@ function cleanup_ecr() {
   done
 }
 
+function validate_env() {
+  local provider=$1
+
+  if [ "$provider" = "aws" ]; then
+    if [[ -z ${AWS_REGION} ]] || [[ -z ${AWS_ACCOUNT_ID} ]]; then
```

Comment:
Let's go with single-bracket conditions here since we really only use single-bracket conditions. That's for keeping it consistent.